### PR TITLE
Main nav

### PR DESCRIPTION
**Pull request by CocoaBean5**
## Asana Ticket(s)
- See ticket https://app.asana.com/0/0/1198894218701637/f

## Description
- Creating main navigation using Layout builder
- Creating haas nav, and limiting visibility via term conditions

## Testing instructions
- `ddev start`, `ddev ssh`, `composer install`
- `drush cim`, `drush cr`
- In theme root, run `npm start`
- Go into Menu, and create menu items for main nav (/admin/structure/menu/manage/main) and haas according to directions seen on https://docs.google.com/document/d/1mlO_wlFxPMV5SJGdBO65urVsUROOn1PmA4ivI_ZraQo/edit#. (Creating New Menu)
- Place the blocks as seen on ^^ doc.
- See that the navigation block shows on the header, and the drop down displays three column menu.
Screen Shot 2020-10-22 at 1 45 38 PM

- See that any page that is tagged with Haas taxonomy, it shows the Haas main menu.
Screen Shot 2020-10-22 at 1 54 13 PM

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -112,6 +114,12 @@
       },
       "drupal/term_condition": {
         "Drupal 9 readiness (https://www.drupal.org/project/term_condition/issues/3133793)": "https://www.drupal.org/files/issues/2020-05-13/3133793-11.patch"
+      },
+      "drupal/menu_item_extras": {
+        "Error when creating new menus (https://www.drupal.org/project/menu_item_extras/issues/3061342)": "https://www.drupal.org/files/issues/2020-04-22/3061342-07-not-found-plugin_0.patch"
+      },
+      "drupal/term_condition": {
+        "Fixes context not loading properly (https://www.drupal.org/project/term_condition/issues/3177999#comment-13876611)": "https://www.drupal.org/files/issues/2020-10-20/3177999-term_condition_update_annotation.patch"
```
Comment (agentrickard) [inline composer.json]: This can e combined with the term_condition patch line above.

Comment (agentrickard) [reply composer.json]: Here's the new patch:

```
      "drupal/term_condition": {
        "Drupal 9 readiness (https://www.drupal.org/project/term_condition/issues/3133793)": "https://www.drupal.org/files/issues/2020-05-13/3133793-11.patch",
        "Fixes context not loading properly (https://www.drupal.org/project/term_condition/issues/3177999)": "https://www.drupal.org/files/issues/2020-10-20/3177999-term_condition_update_annotation.patch",
        "Add option to display block if no terms are present (https://www.drupal.org/project/term_condition/issues/3178234)": "https://www.drupal.org/files/issues/2020-10-21/3178234-term_condition-show-on-empty-3.patch"
      }```

**Review by tboggia — APPROVED** [review]
Works! 
image

**Comment by tboggia** [comment]
Shouldn't composer.lock be .gitignored?

**Comment by agentrickard** [comment]
No. `composer.lock` is what keeps everyone in sync with the same versions. It should always be updated when new items or versions are added to `composer.json`.

**Pull request merged**